week-2/implement/bmi.js

```diff
@@ -13,3 +13,14 @@
 // Given someone's weight in kg and height in metres
 // When we call this function with the weight and height
 // Then it returns their Body Mass Index to 1 decimal place
+
+function bmi(w,h) {
+    h = h * h;
+    let x = w / h;
+  
+  
+    return(x);
```

Comment:
Good work, however, consider using more descriptive variable names. Instead of x, you might use bmiValue or something similar to make your code more readable.